### PR TITLE
fix: change to 8 from 0.8

### PR DESCRIPTION
**Pull request by vm-001**
### Merge instructions

- [x] Please merge after reviewing

### CODE REVIEW

File: content/en/metrics/types.md [hunk 1]
```diff
@@ -183,7 +183,7 @@ For example, suppose you are submitting a HISTOGRAM metric, `request.response_ti
 | Metric Name                                    | Value  | Datadog In-App Type |
 | ---------------------------------------------- | ------ | ------------------- |
 | `request.response_time.histogram.avg`          | `1.88` | GAUGE               |
-| `request.response_time.histogram.count`        | `0.8`  | RATE                |
+| `request.response_time.histogram.count`        | `8`  | RATE                |
 | `request.response_time.histogram.median`       | `2`    | GAUGE               |
```
Comment (neko-dd) [inline content/en/metrics/types.md:187]:
```suggestion
| `request.response_time.histogram.count`        | `8`    | RATE                |
| `request.response_time.histogram.median`       | `2`    | GAUGE               |
```

**Review by neko-dd — APPROVED** [review]

**Comment by buraizu** [comment]
Hi @vm-001, thanks for the PR. We're just validating this internally, and I'll follow up with any updates.

**Comment by buraizu** [comment]
Hi @vm-001, just following up to confirm that the metric value of `0.8` is valid in this context. This is because the metric will be displayed as a RATE in the Datadog App. Since the the `count` of the metric values in the example is 8, this value will be divided by the number of seconds in a flush interval (`10` by default), resulting in `0.8`. You can read more about the behavior of RATE metrics here:
https://docs.datadoghq.com/metrics/types/?tab=rate#metric-types

As such, I'll close this PR for now. Please feel free to reopen the PR if you have any questions.

**Comment by vm-001** [comment]
Thanks for the explanation. @buraizu

**Pull request closed without merging**